openstates/me/people.py

```diff
@@ -73,41 +77,60 @@ def scrape_reps(self):
                 except UnicodeEncodeError:
                     # str typecasting will raise error for non-ascii characters.
                     d[field] = sh.cell(rownum, col_num).value.encode('utf-8').strip()
-                except IndexError:
-                    # This col_num doesn't exist in the sheet.
-                    pass
 
-            district_number = d['district']
+            try:
+                # xlrd reads this value as a float. However we save it as a string.
+                district_number = str(int(float(d['district'])))
```

Comment:
The content of d['district'] is something like '121.0'.
Calling an int() typecast on it will not work, since the string has a decimal. It needs to be first converted to a float, then typecast it to an int, and then back to a str since we save it that way. Hence the complicated syntax.

We could just truncate the last two characters in the string ('.0'), but I'm not sure whether that'll be straightforward to understand if someone's looking at the code sometime in the future. What do you think?

Reply:
Python has some powerful string formatting, check out https://stackoverflow.com/a/15263885

eg: `'{:.0f}'.format(23.214523)`

Reply:
More holistic: https://pyformat.info/

Reply:
Ahh, this is perfect. Didn't think of it, thanks!